components/dashboard/src/AppNotifications.tsx

```diff
@@ -54,7 +55,18 @@ export function AppNotifications() {
                 showIcon={true}
                 className="flex rounded mb-2 w-full"
             >
-                <span>{topNotification}</span>
+                <span>{topNotification}</span> Manage
+                <a
+                    className="gp-link hover:text-gray-600"
+                    href={
+                        notifications.length > 1
+                            ? `${gitpodHostUrl}t/${notifications[notifications.length - 1]}/billing`
```

Comment:
Is it possible to use the `topNotification` variable here? It would avoid us having to do the lookup.

Reply:
Do we actually need the `gitpodHostURL`? I think we could make the URL relative which would avoid having to specify it here

Reply:
> Is it possible to use the `topNotification` variable here?

How do you mean? `topNotification` is the first sentence that includes the team name. The second entry in the array is the `Manage billing` link.

Reply:
Ah, my bad. I misread the contents of those.